src/simulation/people/person/person.py

```diff
@@ -78,6 +86,7 @@ def get_scheduler(self) -> Scheduler:
         return self._scheduler
 
     def get_work_structures(self) -> List[Location]:
+        logger.info(f"{self._name} is getting all work structures")
```

Comment:
I'd modify the text to say "getting all structures {self._name} is working on." so it is more clear which structures are being gotten.

Reply:
completed